client/state/reader/site-blocks/selectors.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Returns true if a site is known to be blocked
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Integer} siteId Site ID
+ * @return {Boolean} Is the specified site blocked?
+ */
+export function isSiteBlocked( state, siteId ) {
+	return !! state.reader.siteBlocks.items[ siteId ];
+}
```

Comment:
Selectors have moved recently, would you mind adding this in `/state/selectors/`? See p4TIVU-5vl-p2 and https://wpcalypso.wordpress.com/devdocs/selectors